packages/tokamak/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -383,9 +398,13 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
         bytes memory _data
     )
         public
-        payable
         metered(_gasLimit)
     {
+        // Lock token in this contract
```

Comment:
@nguyenzung @rlgns98kr Do you think about preventing the users send ETH via `depositTransaction` function by checking `msg.value ` with zero?

Reply:
I think the function is not payable => we don't need to check msg.value

Reply:
@nguyenzung you're right, and we updated `receive` function to prevent the users transfer ETH to this contract directly.